### DISCUSSION TITLE
Assigining a Specific dashboard to a specific user

### DISCUSSION THREAD

**Discussion by rohan0017** [Security-Onion-Solutions/securityonion#12130] [Unsupported Versions]
Is there any way to assign a specific user a specific dashboard in Security onion. When an user logs into SOC, after he clicks kibana he should view his dashboard(my security onion deployment has multiple users I don't want dashboards related to one user visible to other users).

I have tried it using "spaces" concept but everytime a user logs in he can see default space and his newly created space.

**Comment by cm-ops** [comment]
You would have to do that in Spaces.

**Reply by rohan0017** [reply]
Yes I have configured different spaces for each users, but every time they log in to SOC and click on kibana, two spaces are visible they can view dashboards of other users as well.

I want users to view their specific dashboards only.

**Reply by cm-ops** [reply]
Did you create a specifc role to assign to a user and set `"spaces": ["name_of_space"]`.  https://docs.securityonion.net/en/2.4/rbac.html#creating-custom-roles

**Reply by rohan0017** [reply]
Oh ok I will try this("spaces": ["name_of_space"]). Thanks.

Can you please tell me the correct placement of "spaces" in .json file.

I have added it after applications:
{
  "cluster": [],
  "indices": [...],
  "applications": [...],
  "spaces": ["name_of_space"],  <---
  "run_as": []
}

Outcome:
I have created a custom role called as limited-auditortwo. And I assigned the custom space created for that role. When I log in using that role and click on kibana I it is again asking me to select the space. It is not directly taking me to assigned space.
